### PR TITLE
add ability to print default values for all operator variables (#675)

### PR DESCRIPTION
**Pull request by Amper**
issue #675

### CODE REVIEW

File: e2e/suite/suite.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package suite
 
 import (
 	"context"
+	"github.com/VictoriaMetrics/operator/internal/config"
```
Comment (Haleygo) [inline e2e/suite/suite.go]: Need some format here.

**Review by Haleygo — APPROVED** [review]
LGTM!

File: e2e/suite/suite.go (hunk 1 above)
Comment (Amper) [reply e2e/suite/suite.go]: fixed

**Review by Haleygo — APPROVED** [review]
LGTM!

**Pull request merged**